### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by fmuyassarov**
Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,41 @@
-# Metal³ Documentation
+<p align="center"><img alt="Metal³" src="./images/metal3.png" /></p>
 
-![Metal³ Logo](images/metal3.png)
+# Metal³
+
+## Useful links
+
+* [Quick start](URL)
```
Comment (dhellmann) [inline README.md:7]: Does the `URL` part here need to be replaced?

Comment (dhellmann) [reply README.md:7]: Maybe this section should come after the "What is" section?

File: README.md [hunk 2]
```diff
@@ -1,28 +1,41 @@
...
+* [Demos](https://www.youtube.com/watch?v=VFbIHc3NbJo&list=PL2h5ikWC8viKmhbXHo1epPelGdCkVlF16&ab_channel=Metal3)
+* [Blog posts](https://metal3.io/blog/index.html)
+* [FAQ](https://metal3.io/blog/index.html)
+
+## What is the Metal³
```
Comment (dhellmann) [inline README.md:12]:
```suggestion
## What is Metal³
```

File: README.md [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
-
-## More Documentation
-
-For more documentation, please see <http://metal3.io/documentation.html>
-
-[notes]: https://docs.google.com/document/d/1d7jqIgmKHvOdcEmE2v72WDZo9kz7WwhuslDOili25Ls/edit
-[recordings]: https://www.youtube.com/playlist?list=PL2h5ikWC8viJY4SNeOpCKTyERToTbJJJA
+## Community
+
+Metal³ is constantly growing and we are very happy with every single contribution
```
Comment (dhellmann) [inline README.md:21]: I think I understand the intent, but I don't agree with "we are very happy with every single contribution." We need to balance encouraging contributions from newcomers and implying that we'll take whatever code is dumped on us and maintain it indefinitely. Maybe say "we would be happy to collaborate with you"?

File: README.md [hunk 4]
```diff
@@ -1,28 +1,41 @@
...
+  for the project related anouncements, discussions and questions.
+* Come and meet us in our Bi-Weekly community meetings on every alternate
+  Wednesday at 13:00 UTC on [Zoom](https://zoom.us/j/97255696401?pwd=ZlJMckNFLzdxMDNZN2xvTW5oa2lCZz09)
+* If you missed the previous community meeting, you can still find the notes
+  [here](https://docs.google.com/document/d/1d7jqIgmKHvOdcEmE2v72WDZo9kz7WwhuslDOili25Ls/edit)
+  and recordings [here](https://www.youtube.com/playlist?list=PL2h5ikWC8viJY4SNeOpCKTyERToTbJJJA)
+
+## Code of Conduct
+
+See our [Coce of Conduct](https://github.com/metal3-io/metal3-docs/blob/master/CODE_OF_CONDUCT.md)
```
Comment (dhellmann) [inline README.md:41]:
```suggestion
See our [Code of Conduct](https://github.com/metal3-io/metal3-docs/blob/master/CODE_OF_CONDUCT.md)
```

File: README.md (hunk 1 above)
Comment (fmuyassarov) [reply README.md]: Sorry, for the missing URL. Fixed that.

File: README.md (hunk 3 above)
Comment (fmuyassarov) [reply README.md]: Fixed, thanks.

**Pull request merged**